Tests/ApodiniTests/Utilitites/TypeInfoTests.swift

```diff
@@ -26,6 +25,24 @@ class TypeInfoTests: ApodiniTests {
         // A case that should throw an error in isOptional
         XCTAssertEqual(isOptional((() -> Void).self), false)
     }
+
+    func testIsEnum() {
+        /// A custom type
+        enum Test {
+            case unit
+            case integration
+            case system
+        }
+
+        XCTAssertEqual(isEnum(Test.self), true)
+        XCTAssertEqual(isEnum(Int.self), false)
```

Comment:
You could also use the abbreviations `XCTAssertTrue` or `XCTAssertFalse`

Reply:
thanks! I sticked to the pattern inside this test file, as it appears to sometimes be more readable without the short-hand syntax. :)